network/servicenetwork/servicenetwork.go

```diff
@@ -17,12 +17,15 @@
 package servicenetwork
 
 import (
+	"bytes"
+	ecdsa2 "crypto/ecdsa"
 	"io/ioutil"
 	"strings"
 
 	"github.com/insolar/insolar/configuration"
 	"github.com/insolar/insolar/core"
 	"github.com/insolar/insolar/core/message"
+	"github.com/insolar/insolar/cryptohelpers/ecdsa"
```

Comment:
In another files, there is a notation `ecdsa_helper "github.com/insolar/insolar/cryptohelpers/ecdsa"`. ecdsa2 sounds weird